app/services/ui_notifications/hosts/provision.rb

```diff
@@ -0,0 +1,51 @@
+module UiNotifications
+  module Hosts
+    class Provision < Base
+      def deliver!
+        case state
+        when :build_completed
+          host_build
+        end
+      end
+
+      private
+
+      def host_build
+        logger.debug('Host finished provisioning (build is done) - checking for notficiations')
```

Comment:
typo, `notifications`. Also "build is done" reads a bit weird, "build completed", or "Host built completed (provisioning has finished). Checking for notifications" 

IMO this should be `logger.info`, as it is already for importing facts or reports

Since we have the subject, the message is more traceable by using `Host #{subject}` in the message.